packages/trader/src/AppV2/Components/EntryExitDetails/entry-exit-details.tsx

```diff
@@ -0,0 +1,89 @@
+import { Text, CaptionText } from '@deriv-com/quill-ui';
+import CardWrapper from '../CardWrapper';
+import React, { useMemo } from 'react';
+import { TContractInfo, addComma, getEndTime } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+
+const getDateTimeFromEpoch = (epoch?: number) => {
+    if (epoch) {
+        const date = new Date(epoch * 1000);
+        const formattedDate = date.toUTCString().split(' ').slice(0, 4).join(' ');
+        const formattedTime = `${date.toUTCString().split(' ')[4]} GMT`;
```

Comment:
Maybe it's better to use `moment` library for such kind of things?

Reply:
Also, I think for time formatting we have existing function `formatTime` from `@shared`

Reply:
thanks done